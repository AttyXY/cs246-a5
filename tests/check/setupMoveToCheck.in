setup
+ K e1
+ k e5
+ r a2
= black
done
game human human
move a2 a1
quit 
quit
