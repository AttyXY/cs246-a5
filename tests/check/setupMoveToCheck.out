WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  ----k---
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  ----k---
4  --------
3  --------
2  r-------
1  ----K---
   abcdefgh

STARTING GAME!

8  --------
7  --------
6  --------
5  ----k---
4  --------
3  --------
2  r-------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  ----k---
4  --------
3  --------
2  --------
1  r---K---
   abcdefgh

White is in check.
Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
